Installation/CMakeLists.txt

```diff
@@ -39,6 +39,27 @@ option(CGAL_HEADER_ONLY
 
 message("== Setting paths ==")
 
+if (NOT CGAL_DATA_DIR)
+  if(DEFINED ENV{CGAL_DATA_DIR})
+    set(CGAL_DATA_DIR $ENV{CGAL_DATA_DIR})
+  else()
+    if (EXISTS "${CMAKE_SOURCE_DIR}/data")
```

Comment:
Side remark: use `CGAL_SOURCE_DIR` instead of `CMAKE_SOURCE_DIR`, to allow the CGAL repository to be a sub-directory of a bigger CMake project.